help/stats/2019_by_area/index.md

```diff
@@ -0,0 +1,96 @@
+
+Submission data is available in bulk [via OAI](../../oa).
+
+The current submission rates (i.e., averaged over calendar year 2019) as percentages of the average 2019 [overall](http://arxiv.org/stats/monthly_submissions) average rate of 12989/month (155,866 total for 2019) are roughly:
+
+*   [cs](#cs_yearly) (43317): 27.8%
+*   [math+math-ph](#math_yearly) (35107): 22.5%
+*   [cond-mat](#cond-mat_yearly) (15216): 9.8%
+*   [astro-ph](#astro-ph_yearly) (14420): 9.3%
+*   [physics](#physics_yearly) (13238): 8.5%
+*   [hep](#hep_yearly) (9687): 6.2%
+*   [eess](#eess_yearly) (6082): 3.9%
+*   [quant-ph](#quant-ph_gr-qc_yearly) (5557): 3.6%
+*   [stat](#stat_yearly) (4335): 2.8%
+*   [gr-qc](#quant-ph_gr-qc_yearly) (3045): 2.0%
+*   [q-bio](#q-bio_yearly) (1853): 1.2%
+*   [nucl](#nucl_yearly) (1720): 1.1%
+*   [nlin](#nlin_yearly) (826): 0.5%
+*   [q-fin](#q-fin_yearly) (810): 0.5%
+*   [econ](#econ_yearly) (671): 0.4%
+
+![Cumulative submissions in each subject area by year](cumsubs.png)
+
+The graphs above show the **cumulative** data as a function of time, where the current (2019) cumulative number of submissions to date as percentages of the total 1,639,224 submissions are roughly:
+
+*   [math+math-ph](#math_yearly) (367639): 22.4%
+*   [cond-mat](#cond-mat_yearly) (240761): 14.7%
+*   [hep](#hep_yearly) (232190): 14.2%
+*   [astro-ph](#astro-ph_yearly) (233924): 14.3%
+*   [cs](#cs_yearly) (211616): 12.9%
+*   [physics](#physics_yearly) (114414): 7.0%
+*   [quant-ph](#quant-ph_gr-qc_yearly) (73863): 4.5%
+*   [gr-qc](#quant-ph_gr-qc_yearly) (47042): 2.9%
+*   [nucl](#nucl_yearly) (37524): 2.3%
+*   [stat](#stat_yearly) (26085): 1.6%
+*   [nlin](#nlin_yearly) (19002): 1.2%
+*   [q-bio](#q-bio_yearly) (18024): 1.1%
+*   [eess](#eess_yearly) (8819): 0.5%
+*   [q-fin](#q-fin_yearly) (7137): 0.4%
+*   [econ](#econ_yearly) (1184): 0.1%
+
+Graphs of yearly submission rates by subject areas
+--------------------------------------------------
+
+![all physics submissions by year](phys_yearly.png)
+
+<span id="cs_yearly"></span>
```

Comment:
The anchors are rendered inline with the graphics which means and anchor link takes users to below the graphic they are looking for. This appears to be a problem with the 2018 template. Perhaps an extra line break between the span and the figure would work. In the future we may want to give text titles to the figures for improved accessibility.

Reply:
Thanks, I'll work on a fix.

Reply:
@jimentwood your suggestion worked like a charm!